### DISCUSSION TITLE
Please Rename NickToons Into NickToons Pluto TV Canada

### DISCUSSION THREAD

**Discussion by Westh977** [iptv-org/community#1161] [Q&A]
Hello, There's 2 NickToons From M3U IPTV Playlist it's Play's Nicktoons Pluto TV In English In Canada It Launched In December 2023 I want you Change IT Please
https://static.wikia.nocookie.net/ideas/images/e/e2/Modern-splat.png/revision/latest?cb=20240415231352

**Comment by freearhey** [comment]
https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-channel-description
